docs/core/tools/dotnet-workload-sets.md

```diff
@@ -0,0 +1,165 @@
+---
+title: .NET SDK workload sets
+description: "Learn how to use workload sets to control which versions of workloads get installed, updated, or restored."
+author: tdykstra
+ms.author: tdykstra
+ms.service: dotnet
+ms.topic: how-to
+ms.date: 08/02/2024
+
+#customer intent: As a developer using optional workloads, I want to keep my workload versions in sync so that I can avoid conflicts betwween versions. I also want to avoid unexpected workload updates, so that I can avoid disruptions in development.
+---
+<!--https://aka.ms/patterns-feedback-->
+
+# .NET SDK workload sets
+
+The *workload sets* feature provides a version number that represents a group of .NET SDK workloads. The [install](dotnet-workload-install.md), [update](dotnet-workload-update.md), and [restore](dotnet-workload-restore.md) commands use this number in *workload-set update mode* to provide the following benefits:
+
+* You control the cadence of change for installed workload versions. The alternative mode of operation without using workload sets is called *loose manifests update mode*. In this mode, workloads are updated automatically as new versions of individual workloads are released onto any configured NuGet feeds. In workload set update mode, workloads stay at a specific workload set version until you explicitly change that version.
+* You can install and update a combination of workload versions that ship at the same time and are known to work together.
+* You can ensure that everyone on your team is always working on the same workload versions.
+* You don't have to use a rollback file to specify what workload version you want to be on.
```

Comment:
Workload sets are opt-in in 8.0.4xx versions of the .NET SDK but will become opt-out in a future release (we hope 9.0.1xx but we can't commit to that so not sure how to phrase it).

Reply:
I cover this difference in the Prerequisites section. Is it certain enough that 9.0.1xx will be opt-out that we can say that, and go back and change it if it turns out to not be accurate? Or better to just say it's opt-in and not discuss the possibility of opt-out happening until it happens?

Reply:
It's not yet certain that 9.0.100 will be opt-out, so what you have in the Prereqs seems fine to me. We can always clarify once we have a decision made.